### ISSUE TITLE
The change of environment variables are applied to puma worker processes

### ISSUE THREAD

**Issue by kzkn** [ajgon/opsworks_ruby#284]
### Expected Behavior
The change of environment variables are applied to puma worker processes after restarted them.

### Actual Behavior
The change of environment variables are not applied to puma worker processes after restarted them.

Here is an example (rails):

```ruby
class HelloController < ApplicationController
  def show
    render html: "Hello, ENV['YOUR_NAME']"
  end
end
```

Set `YOUR_NAME` as `alice` and deploy, when I open browser, it shows `Hello, alice`.
Then change the value of `YOUR_NAME` to `bob` and deploy, when I open browser, it shows `Hello, alice`. But we expect `Hello, bob` is shown.

### Custom JSON
```json
"appserver": {
    "dot_env": true,
    "worker_processes": 2
},
"worker": {
    "adapter": "delayed_job",
    "process_count": 1
}
```

**Comment by kzkn** [comment]
This issue is similar with #281 but it is not same. #281 is workaroundable with deploy twice or `monit restart puma_app`, but this issue is not.
The workaround I know is to `sudo monit stop puma_app` and then `sudo monit start puma_app`. This command will restart the puma main process with new environment variables.

**Comment by ajgon** [comment]
@kzkn I assume, this also can be closed thanks to #282 (including your patch) ? :)

**Comment by kzkn** [comment]
@ajgon Yes, I had checked this issue is solved with #282
thanks!
